cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -93,13 +94,11 @@ module cosmos 'br/public:avm/res/document-db/database-account:0.8.1' = {
       virtualNetworkRules: []
       publicNetworkAccess: 'Enabled'
     }
-    {{- if .DbCosmosMongo.DatabaseName}}
```

Comment:
database name should always be exist. Delete it to keep it align with other DB like mysql.